src/engine/openspaceengine_lua.inl

```diff
@@ -311,6 +312,9 @@ int createSingeColorImage(lua_State* L) {
     colorDict.setValue(key, d);
     TestResult res = DoubleVector3Verifier()(colorDict, key);
 
+    // @TODO (emmbr 2020-02-04) A 'ColorVerifier' would be really useful here, to easily
```

Comment:
Behind Door number three you should find a:
`InRangeVerifier<DoubleVector3Verifier>(glm::dvec3(0.0), glm::dvec3(1.0)`

(i think at least)

Reply:
Would have been nice, but sadly, nope

https://github.com/OpenSpace/OpenSpace/blob/91612f4bfd54f9e241b599e67f9a60b55cdbfb6e/include/openspace/documentation/verifier.h#L728

Reply:
And checking every value individually seemed like too much of a hassle

Reply:
Agree..  Seems like that ought to be possible (https://github.com/OpenSpace/OpenSpace/issues/1491)